### ISSUE TITLE
Tab title not read by screen reader

### ISSUE THREAD

**Issue by carlos-zamora** [microsoft/terminal#9254]
# Environment

```none
Windows Terminal version (if applicable): 1.7.501.0
```
NVDA

# Steps to reproduce

1. Open NVDA
2. Click on an open terminal tab (focus should be there)
3. Move focus to another tab

# Expected behavior

Screen reader reads "Command Prompt tab 2 of 4" (format: <tab title> tab <position> of <size>)

NOTE: this is the behavior for v1.5.10411.0

# Actual behavior

Screen reader reads "tab selected 2 of 4" (or whatever position/length it has).

**Comment by DHowett** [comment]
This is probable fallout from TabHeaderControl and will *need* to be fixed before 1.6 can enter stable.
